CRM/Financial/BAO/Order.php

```diff
@@ -940,6 +940,16 @@ public function setLineItem(array $lineItem, $index): void {
     if ($this->getOverrideTotalAmount()) {
       $this->addTotalsToLineBasedOnOverrideTotal((int) $lineItem['financial_type_id'], $lineItem);
     }
+    elseif (isset($lineItem['line_total_inclusive'])) {
+      if (isset($lineItem['tax_amount_override'])) {
+        // @todo deprecate this - work with Karin to make sure she is not using order api in
+        // a way that deprecating this causes problems.
+        $lineItem['line_total'] = $lineItem['line_total_inclusive'] = $lineItem['tax_amount'];
```

Comment:
@eileenmcnaughton should this be `$lineItem['tax_amount_override']` on the right? Currently if i submit a webform with a taxable financial type, it creates a contribution with amount = $0. Probably because line_total is not set in this step. A quick way to replicate is execute this API -

```
$result = civicrm_api3('Order', 'create', [
  'contact_id' => 202,
  'financial_type_id' => "Member Dues",
  'total_amount' => 220,
  'tax_amount' => 20,
]);
```
where member dues is a taxable ft of 10%.